src/api/document/common.rst

```diff
@@ -1167,7 +1167,7 @@ for the target document by appending the ``rev`` parameter to the
 
 **Request**:
 
-.. code-block:: http
+.. code-block:: none
```

Comment:
We do have it supported in [local lexer](https://github.com/apache/couchdb-documentation/blob/master/ext/httpdomain.py#L629). May be worth to switch onto in instead of doing [fallback](https://github.com/apache/couchdb-documentation/blob/master/ext/httpdomain.py#L653-L656)?